src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -81,6 +81,10 @@ function SidebarLinksData({isFocused, allReportActions, betas, chatReports, curr
         return reportIDsRef.current || [];
     }, [allReportActions, betas, chatReports, currentReportID, policies, priorityMode, isLoading]);
 
+    const currentReportIDRef = useRef(currentReportID);
+    currentReportIDRef.current = currentReportID;
```

Comment:
Is this value assignment needed? currentReportID is set on the line above as the initial value